Directory.Build.targets

```diff
@@ -200,7 +200,9 @@ Expected to exist:
     </ItemGroup>
 
     <Copy SourceFiles="@(AndroidSdkArtifacts)" DestinationFiles="@(AndroidSdkArtifacts->'$(SentryAndroidArtifactsDestination)%(RecursiveDir)%(Filename)%(Extension)')" />
-    <Exec WorkingDirectory="$(SentryAndroidRoot)" Command="cp sentry/build/libs/sentry*.jar $(SentryAndroidArtifactsDestination)sentry.jar" />
+
+    <Copy SourceFiles="$(SentryAndroidRoot)/sentry/build/libs/sentry*.jar"
```

Comment:
Wildcards don't work with that task do it?

Reply:
do we need the wildcard? is the file name not known?

Reply:
The source name contains a version number while we want the target without that.